### DISCUSSION TITLE
How to add DocSearch in Doks

### DISCUSSION THREAD

**Discussion by iqlal** [thuliteio/doks#707] [Q&A]
Hi, please provide me guide to add DocSearch feature in Doks

**Comment by h-enk** [comment]
I'm planning to write a little guide on it  (but I'm way to busy).

For now, check how it's done in the code of the [getdoks.org repo](https://github.com/h-enk/getdoks.org), it follows the [DocSearch instructions](https://docsearch.algolia.com/docs/DocSearch-v3)
